app/src/main/java/com/imashnake/animite/features/MainActivity.kt

```diff
@@ -124,5 +99,15 @@ fun MainScreen(modifier: Modifier = Modifier) {
         ) {
             NavigationBar(navController = navController)
         }
+
+        Search(
+            modifier = Modifier
+                .align(Alignment.BottomEnd)
+                .padding(
+                    start = dimensionResource(R.dimen.large_padding),
+                    end = dimensionResource(R.dimen.large_padding)
+                ),
+            navController = navController
```

Comment:
Can we lift navController out of SearchBar while we're here? Passing a lambda can achieve the same thing while making it slightly easier to change nav framework in the future